Source/Orts.Simulation/Common/Commands.cs

```diff
@@ -2042,4 +2158,155 @@ public override string ToString()
             return base.ToString();
         }
     }
+
+    // EOT Commands
+
+    [Serializable()]
+    public sealed class EOTCommTestCommand : Command
+    {
+        public static MSTSLocomotive Receiver { get; set; }
+
+        public EOTCommTestCommand(CommandLog log)
+            : base(log)
+        {
+            Redo();
+        }
+
+        public override void Redo()
+        {
+            if (Receiver?.Train?.EOT != null)
+            {
+                Receiver.Train.EOT.CommTest();
+            }
```

Comment:
```suggestion
            Receiver?.Train?.EOT?.CommTest();
```
I think this simplified syntax is valid.

Reply:
I believe I had a problem with that. However I've tried again and it seems to work.